chapter2/scenarios/06_Dwarven_Doors.cfg

```diff
@@ -513,15 +513,15 @@
     {NOTRAIT_UNIT 2 Swordsman 7 7}
     [message]
       type=Swordsman
-      message=_"Are you finished with the armors?"
+      message=_"Are you finished with the armour suits?"
     [/message]
     [message]
       type=Dwarvish Runesmith
       message=_"Yes, I am."
     [/message]
     [message]
       type=Swordsman
-      message=_"Ok let me see. (examining armors) They are very strong and nice."
+      message=_"Ok let me see. (examining armour) They are very strong and nice."
```

Comment:
The idea is that he made several suits, not just one. Oh, and I  think that armor is correct in American(?) English.